source/connect/mysql.txt

```diff
@@ -45,25 +66,64 @@ password after the command has been entered.
 
    mysql '--user=reportsUser?source=admin' --default-auth=mongosql_auth -p
 
-This example assumes that the authentication plugin file
-``mongosql_auth.so`` is located in the default MySQL plugin folder.
-The location of the plugin folder varies by platform, but you can
-locate it by running the following command:
+.. note::
+
+   This example assumes that the authentication plugin file
+   ``mongosql_auth.so`` is located in the default MySQL plugin folder.
+   The location of the plugin folder varies by platform, but you can
+   locate it by running the following command:
+
+   .. cssclass:: copyable-code
+   .. code-block:: sh
+
+      mysql_config --plugindir
+
+   ``mysql_config.pl`` can find the plugin directory only on macOS and 
```

Comment:
Reading this, I'm unclear what ``mysql_config.pl`` actually is since it's not referenced anywhere else on the page. I'm assuming that it is the service (pardon if 'service' is the incorrect terminology) the mysql_config command in the code block implements. Can we make that clearer by saying something to the effect of:

> ``mysql_config.pl``, which implements the above command, can find the plugin directory....